srcs/incs/Utils.hpp

```diff
@@ -12,6 +12,7 @@
 
 std::string	TrimString(const std::string &str, const std::string &trim_chars);
 std::string	ToLowerString(std::string str);
+std::string	ToUpperString(std::string str);
 char		*DuplicateString(const std::string &str);
 std::string	DecodeUrl(const std::string &encoded_url);
 std::string	PathExtension(const std::string &path);
```

Comment:
As mentioned, it feels like maybe we could get rid of this one, but we have to close this project at some point...